src/ffi3/object.rs

```diff
@@ -594,7 +634,7 @@ pub struct PyType_Slot {
 
 impl Default for PyType_Slot {
     fn default() -> PyType_Slot {
-        unsafe { mem::zeroed() }
+        unsafe { ::std::mem::zeroed() }
```

Comment:
Is this required?

Reply:
nope, removed :) 
I think this was a hiccup from some old version.